ts/src/coinbase.ts

```diff
@@ -1318,6 +1318,9 @@ export default class coinbase extends Exchange {
             const id = this.safeString (currency, 'id', key);
             const name = this.safeString (currency, 'name');
             const code = this.safeCurrencyCode (id);
+            const lastCurrencyInformations = this.safeValue (this.currencies ?? {}, code, {});
+            // Precision is not provided by this endpoint but when we fetch balances - if we fetched it once, we use it since it won't change
+            const precision = this.safeValue (lastCurrencyInformations, 'precision', undefined);
```

Comment:
didn't understand: at the first fetch, `this.currencies` is not yet defined, right? Could you also reformulate the comment to make it clearer?

Reply:
```
            // We do not have the precision property here. We will fetch it when we fetch balances (fetchBalance) and populate the currencies object
            // Because we only need this precision for withdrawals, it's ok since we will fetch balances at least once before perfoming the withdrawal
            // But because fetchCurrencies can be called at any time, we don't want to totally replace the currencies object and lose the precision property
            // We use the last currency informations to keep the precision property (since it won't change between two calls
```